ios/AdyenDropInModule.swift

```diff
@@ -112,6 +112,74 @@ class AdyenDropInModule: NSObject {
         }
     }
     
+    @objc
+    func setSubmitCallback(_ onSubmit: @escaping RCTResponseSenderBlock) {
+        print("Called setSubmitCallback")
+        self.onSubmitCallback = onSubmit
```

Comment:
Looks like
`private var onSubmitCallback: RCTResponseSenderBlock? = nil`
and
`private var onAdditionalDetailsCallback: RCTResponseSenderBlock? = nil`
are missing from the PR

Reply:
Good catch! It looks like those parts fell off somewhere along the way... Updated now.